server/index.js

```diff
@@ -69,6 +69,15 @@ io.on('connection', (socket) => {
         });
         callback();
     });
+
+    //received signal to end the session with no matches, emit redirect signal to all guests and host
+    socket.on('noMatch', (room, callback) => {
+        console.log('no match signal received')
```

Comment:
I think we should remove the `console.log()` lines in production. What do you think?